extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -525,6 +525,18 @@ describes.realWin(
         });
       });
 
+      it('should contain aanf if sticky ad', () => {
+        const ampStickyAd = createElementWithAttributes(doc, 'amp-sticky-ad', {
+          'layout': 'nodisplay',
+        });
+        element.setAttribute('data-no-fill', 'true');
+        ampStickyAd.appendChild(element);
+        doc.body.appendChild(ampStickyAd);
+        return impl.getAdUrl().then(adUrl => {
+          expect(adUrl).to.contain('aanf=true');
```

Comment:
Ideally you would test that it starts/ends with query parameter delimiters.   Also you can put the promise into the expectation.  Change to:

return expect(impl.getAdUrl()).to.eventually.match(/(\?|&)aanf=true(&|$)/);

Reply:
Done.